### PR TITLE
Add compliance to gdpr law (psgdpr module)

### PR DESCRIPTION
**Pull request by Joukz**
Add compliance with the module psgdpr. If you want to test it, you need to install the official gdpr module by PrestaShop (psgdpr).

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <module>
 	<name>mailalerts</name>
 	<displayName><![CDATA[Mail alerts]]></displayName>
-	<version><![CDATA[3.6.1]]></version>
+	<version><![CDATA[3.6.2]]></version>
```
Comment (eternoendless) [inline config.xml]: Since you are adding a feature, you should update the version to 3.7.0

File: mailalerts.php [hunk 2]
```diff
@@ -48,7 +48,7 @@ public function __construct()
 	{
 		$this->name = 'mailalerts';
 		$this->tab = 'administration';
-		$this->version = '3.6.1';
+		$this->version = '3.6.2';
```
Comment (eternoendless) [inline mailalerts.php]: Again, 3.7.0

File: upgrade/install-3.6.2.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2016 PrestaShop SA
+ * @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+if (!defined('_PS_VERSION_'))
+	exit;
+
+function upgrade_module_3_6_2($object)
```
Comment (eternoendless) [inline upgrade/install-3.6.2.php]: Again, 3_7_0

**Comment by prasanthSelvar** [comment]
Tested and OK! :+1:

**Comment by eternoendless** [comment]
Thank you @Joukz

**Comment by mhd6** [comment]
Hi, Thank you my friend..!

**Comment by Eolia** [comment]
The hook in the Product.tpl must be:
`{hook h='displayGDPRConsent' mod='psgdpr' id_module=$id_module}`

**Comment by Eolia** [comment]
And find('#gdpr_consent') is a very bad idea if multiple compliant modules are on the same page...

**Comment by Quetzacoalt91** [comment]
Yeah, good point about `#gdpr_consent`.

However, I asked to remove `mod='psgdpr'` in order to let other modules using it. Why should we bring it back?

**Comment by anto7880** [comment]
If I uninstall email alert I do not receive email notifications for new orders .... why does this happen?

**Comment by Eolia** [comment]
Enable and configure ps_emailalerts^^

**Comment by anto7880** [comment]
In the list of modules I do not see ps_mailalert, I see the folder only in ftp :(

**Comment by anto7880** [comment]
Where can I find the ps_mail alert module download?

**Comment by anto7880** [comment]
thank you so much!!

**Pull request merged**